lib/ssl/src/ssl_internal.hrl

```diff
@@ -270,6 +271,21 @@
 				{stop, any(), any()}.
 -type ssl_options()          :: map().
 
+
+-define(SSL_LOG(Level, Descr, Reason),
+        fun() ->
+                case get(log_level) of
+                    undefined ->
```

Comment:
Will not log_level always be defined ? We default to notice! And if you want no logs it should be none!

Reply:
Because a function with the macro can be called from a process that have not set the log_level.
Such as in the testcases below :-)

It is also done to make sure we don't crash if we later wrongly call the macro from such a process.